release-notes/opensearch-sql.release-notes-3.4.0.0.md

```diff
@@ -5,6 +5,8 @@ Compatible with OpenSearch and OpenSearch Dashboards version 3.4.0
 ### Features
 * Support `chart` command in PPL ([#4579](https://github.com/opensearch-project/sql/pull/4579))
 * Support `Streamstats` command with calcite ([#4297](https://github.com/opensearch-project/sql/pull/4297))
+* Support `multisearch` command in calcite ([#4332](https://github.com/opensearch-project/sql/pull/4332))
+* Add replace command with Calcite ([#4451](https://github.com/opensearch-project/sql/pull/4451))
```

Comment:
Keep the message consistent with other features.
Change it to support `replace`